persistence/src/commonMain/db_user/com/wire/kalium/persistence/1.sqm

```diff
@@ -0,0 +1,92 @@
+DROP VIEW ConversatonDetails;
```

Comment:
Should we name this `1.sqm` or something more descriptive?

Reply:
`1.sqm` means "migration from verstion 1 to version 2", so it has to be `1` :)